src/one_dragon/base/push/push_config.py

```
@@ -33,7 +45,38 @@ def __init__(self):
+                    'qywx_app_agent_id': parts[3]
+                }
+                # 可选的media id
+                if len(parts) >= 5:
+                    migration_map['qywx_app_media_id'] = parts[4]
+
+                for new_key, new_value in migration_map.items():
+                    # 只有当新key不存在或为空时，才进行迁移，避免覆盖用户的新设置
+                    if not self.data.get(new_key):
+                        self.data[new_key] = new_value
+
+            # 迁移完成，删除旧key并保存
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)